docs/studios/api_reference/authentication.mdx

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_label: 'Authentication'
+---
+
+# Authenticating with API Keys
+
+The Regression Games platform provides the ability to generate API keys, which can be used
+to authenticate against our API. This is the recommended way to authenticate with our API
+when connecting external systems to Regression Games.
+
+## Generating an API Key
+
+An API key can be generated from your [account](https://play.regression.gg/account) page. At
+the bottom of the screen within the **API Keys** section, enter a unique name for your key, and then
+click **Generate New Key**. This token should be copied and stored somewhere safe - you won't be 
+able to see it again after it is generated. Keys have a default expiration date of 1 year.
```

Comment:
This should be common sense, but might be a good idea emphasize the "store somewhere safe" part as a warning or danger admonition

Reply:
Thanks! Added this as well.